okio/src/commonMain/kotlin/okio/internal/RealBufferedSource.kt

```diff
@@ -418,19 +441,18 @@ internal inline fun RealBufferedSource.commonRangeEquals(
 ): Boolean {
```

Comment:
Also reimplement `BufferedSource.rangeEquals()` on `BufferedSource.indexOf()`. Of course.